src/moderngl.cpp

```diff
@@ -273,11 +273,12 @@ struct MGLTextureCube {
     int texture_obj;
     int width;
     int height;
-    int depth;
+    bool depth;
     int components;
     int min_filter;
     int mag_filter;
     int max_level;
+	int compare_func;
```

Comment:
please replace tabs with 4 spaces.
with vscode just press `ctrl`+`shift`+`p` then type `>Convert Indentation to Spaces` should do the trick